src/test/java/org/broadinstitute/hellbender/engine/FeatureInputUnitTest.java

```diff
@@ -257,4 +257,24 @@ public void testToString() {
         Assert.assertEquals(namelessGenomicsDB.toString(), "gendb://" + new File("file1").getAbsolutePath(), "String representation of nameless FeatureInput with genomicsDB path incorrect");
         Assert.assertEquals(namedGenomicsDB.toString(), "name:gendb://" + new File("file1").getAbsolutePath(), "String representation of named FeatureInput with genomicsDB path incorrect");
     }
+
+    @DataProvider(name = "IsUserSuppliedNameData")
+    public Object[][] isUserSuppliedNameData() {
+        return new Object[][] {
+                // input String, expected Feature path, expected logical name
```

Comment:
Comment here is not accurate -- there are only two values per test case, not three.

Reply:
removed